app/javascript/components/widgets/widgets/forest-change/tree-loss/initial-state.js

```diff
@@ -39,7 +39,7 @@ export const initialState = {
     },
     sentences: {
       initial:
-        'From {startYear} and {endYear}, {location} lost {loss} of tree cover, equivalent to a {percent} decrease since {extentYear} and {emissions} of CO\u2082 emissions.',
+        'From blah thing {startYear} and {endYear}, {location} lost {loss} of tree cover, equivalent to a {percent} decrease since {extentYear} and {emissions} of CO\u2082 emissions.',
```

Comment:
this is a mistake, right? 😂

Reply:
lol my bad, I had a pretty high fever when I coded that  🤒